src/components/Navbar/NavBar.tsx

```diff
@@ -220,6 +220,15 @@ const HeaderContainer = styled.header`
     `
     margin-top: 40px;
   `}
+
+  @media (max-width: 600px) {
```

Comment:
I think we have directives for standard widths, e.g. `tablet-width` somewhere if you want to grab that